docs/advanced/backups.md

```diff
@@ -2,6 +2,12 @@
 
 ## Creating backups
 
+There's two kinds of backups.
+
+Overrides and .env get backed up to ~/.config/.compose.backups which is a hidden folder (if you don't see it you need to list hidden folders). This happens automatically and is not optional.
```

Comment:
`Overrides and .env` => `Your .env and docker-compose.override.yml file (if you have one)`